sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -101,6 +101,18 @@ def test_feature_importances():
     mask = np.abs(transformer.estimator_.coef_) > 1e-5
     assert_array_equal(X_new, X[:, mask])
 
+    # Linear model with CSR yields same result as with ndarray
+    lest = LinearSVC()
```

Comment:
I don't think that would be necessary, just that i've never seen `lest` used before.